hyp3_gamma/metadata/templates/insar/readme.md.txt.j2

```diff
@@ -211,9 +211,9 @@ The DEM used for this product is {{ dem_name }}, which has a native pixel spacin
 ----------------
 ## 11. Water Mask
 
-The water mask identifies coastal waters and major inland waterbodies beyond 3 km of the shoreline. These areas are assigned a pixel value of 0. All remaining pixels (land, small inland waterbodies, land ice, and near-shore coastal areas) have a value of 1. The water mask is stored as an 8-bit unsigned integer GeoTIFF file and is always included in the product package.
+The water mask identifies coastal waters and major inland waterbodies. These areas are assigned a pixel value of 0. All remaining pixels (land, small inland waterbodies, and land ice) have a value of 1. The water mask is stored as an 8-bit unsigned integer GeoTIFF file and is always included in the product package.
 
-The water mask is not precise; the [GSHHG](http://www.soest.hawaii.edu/wessel/gshhg/) L1, L2 and L5 shoreline data is buffered out 3 km to reduce the possibility of near-shore features being excluded while still being a useful tool for masking out phase unwrapping artifacts over water.
+The water mask is not precise, however the [GSHHG](http://www.soest.hawaii.edu/wessel/gshhg/) L1, L2 and L5 shoreline data is  a useful tool for masking out phase unwrapping artifacts over water.
```

Comment:
Per @hjkristenson 's notes, the new shapefile now includes L3 as well, so I think we need to add it to the list here